.github/workflows/conan.yml

```diff
@@ -18,17 +18,6 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v4
 
-      - name: Get new commit count
-        id: new_commit_count
```

Comment:
I think it's just a bit redundant/unnecessary, I would like to get the new commit count thing working correctly but I think the correct logic might actually be something like "did the last build fail, or have there been any new commits since the last successful build" which escapes me a bit as to how to express that in github actions

Reply:
not sure how helpful this is but I found this online: https://github.com/marketplace/actions/last-successful-commit 

I am not super familiar with github actions but maybe you could use this to get the SHA of the last successful commit, then get the SHA of the latest commit in the branch and compare them. If they match then there have been no new commits since the last successful build, otherwise there have been new commits?

Reply:
I will give that a shot!

Reply:
@jckras ended up giving it a shot with `last-successful-commit`, I _think_ this is how it should work but probably won't really know until a scheduled build tries to run